diff --git a/CMakeLists.txt b/CMakeLists.txt
index b72b116..241a32d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -120,19 +120,9 @@ IF (${sparsehash} STREQUAL "sparsehash-NOTFOUND")
 ENDIF()
 
 #LZ4
-# find_library(lz4 lz4)
-# find_path(lz4h lz4.h)
-# IF (${lz4h} STREQUAL "lz4h-NOTFOUND")
-#     message("Could not find LZ4. I'm going to use the version from kognac")
-    message("I'm going to use LZ4 the version from kognac")
-    include_directories(${KOGNAC_LIB}/external/lz4/lib/)
-    LINK_DIRECTORIES(${KOGNAC_LIB}/external/lz4/lib)
-    add_library(lz4 STATIC IMPORTED)
-    set_property(TARGET lz4 PROPERTY IMPORTED_LOCATION ${KOGNAC_LIB}/external/lz4/lib/${CMAKE_STATIC_LIBRARY_PREFIX}lz4${CMAKE_STATIC_LIBRARY_SUFFIX})
-# ELSE()
-# include_directories(lz4h)
-#    message("-- Found LZ4")
-# ENDIF()
+find_library(lz4 ${CMAKE_STATIC_LIBRARY_PREFIX}lz4${CMAKE_STATIC_LIBRARY_SUFFIX})
+find_path(lz4h lz4.h)
+include_directories(lz4h)
 
 IF (JAVA)
     find_package(Java REQUIRED)
