From bef7c75789879a55479a2084a7089550628d0d48 Mon Sep 17 00:00:00 2001
From: Maximilian Marx <mmarx@wh2.tu-dresden.de>
Date: Fri, 2 Sep 2022 01:37:23 +0200
Subject: [PATCH] Patch lz4

Signed-off-by: Maximilian Marx <mmarx@wh2.tu-dresden.de>
---
 CMakeLists.txt | 17 +++--------------
 1 file changed, 3 insertions(+), 14 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 289b2f3..9cc598c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -127,20 +127,9 @@ IF (${sparsehash} STREQUAL "sparsehash-NOTFOUND")
     include_directories(${KOGNAC_LIB}/external/sparsehash/src/)
 ENDIF()
 
-#LZ4
-# find_library(lz4 lz4)
-# find_path(lz4h lz4.h)
-# IF (${lz4h} STREQUAL "lz4h-NOTFOUND")
-#     message("Could not find LZ4. I'm going to use the version from kognac")
-    message("I'm going to use LZ4 the version from kognac")
-    include_directories(${KOGNAC_LIB}/external/lz4/lib/)
-    LINK_DIRECTORIES(${KOGNAC_LIB}/external/lz4/lib)
-    add_library(lz4 STATIC IMPORTED)
-    set_property(TARGET lz4 PROPERTY IMPORTED_LOCATION ${KOGNAC_LIB}/external/lz4/lib/${CMAKE_STATIC_LIBRARY_PREFIX}lz4${CMAKE_STATIC_LIBRARY_SUFFIX})
-# ELSE()
-# include_directories(lz4h)
-#    message("-- Found LZ4")
-# ENDIF()
+find_library(lz4 ${CMAKE_STATIC_LIBRARY_PREFIX}lz4${CMAKE_STATIC_LIBRARY_SUFFIX})
+find_path(lz4h lz4.h)
+include_directories(lz4h)
 
 IF (JAVA)
     find_package(Java REQUIRED)
-- 
2.36.2

